httpcore5-h2/src/main/java/org/apache/hc/core5/http2/ssl/H2TlsSupport.java

```diff
@@ -46,6 +46,10 @@ public static void setEnableRetransmissions(final SSLParameters sslParameters, f
         ReflectionUtils.callSetter(sslParameters, "EnableRetransmissions", Boolean.TYPE, value);
     }
 
+    /**
+     * @deprecated Use {@link SSLParameters#setApplicationProtocols(String[])}.
+     */
+    @Deprecated
     public static void setApplicationProtocols(final SSLParameters sslParameters, final String[] values) {
         ReflectionUtils.callSetter(sslParameters, "ApplicationProtocols", String[].class, values);
```

Comment:
Any reason not to update this call to `sslParameters.setApplicationProtocols(values);`?